.evergreen/config.yml

```diff
@@ -172,6 +176,118 @@ functions:
           ATLAS_REPL_SRV='${ATLAS_REPL_SRV}' ATLAS_SHRD_SRV='${ATLAS_SHRD_SRV}' ATLAS_FREE_SRV='${ATLAS_FREE_SRV}' ATLAS_TLS11_SRV='${ATLAS_TLS11_SRV}' ATLAS_TLS12_SRV='${ATLAS_TLS12_SRV}' \
           sh ${PROJECT_DIRECTORY}/.evergreen/run-atlas-tests.sh
 
+  "run ocsp test":
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+
+          MONGODB_URI="${MONGODB_URI}" \
+          SWIFT_VERSION=${SWIFT_VERSION} \
+          OCSP_TLS_SHOULD_SUCCEED="${OCSP_TLS_SHOULD_SUCCEED}" \
+          OCSP_ALGORITHM="${OCSP_ALGORITHM}" \
+            bash ${PROJECT_DIRECTORY}/.evergreen/run-ocsp-test.sh
+
+  run-valid-ocsp-server:
```

Comment:
these functions and all the task definitions were taken pretty much straight from the Rust config, which were taken pretty much straight from the Go config.

Reply:
♻️